db_models/event_user_model.py

```diff
@@ -20,10 +20,9 @@ class EventUser_DB(BaseModel_DB):
 
     event: Mapped["Event_DB"] = relationship(back_populates="event_users")
     event_id: Mapped[int] = mapped_column(ForeignKey("event_table.id"), primary_key=True)
-    
-    priority: Mapped[str] = mapped_column(default = "Övrigt")
 
-    created_at: Mapped[datetime] = created_at_column()
-    latest_modified: Mapped[datetime] =  latest_modified_column()
+    priority: Mapped[str] = mapped_column(default="Övrigt")
+    group_name: Mapped[str] = mapped_column(default="")

```

Comment:
Make group name a nullable column instead of default to empty string. `Mapped[Optional[str]]`